src/jsdoc_transformer.ts

```diff
@@ -616,7 +616,20 @@ export function jsdocTransformer(
               // getOriginalNode(decl) is required because the type checker cannot type check
               // synthesized nodes.
               const typeStr = moduleTypeTranslator.typeToClosure(ts.getOriginalNode(decl));
-              localTags.push({tagName: 'type', type: typeStr});
+              // If @define is present then do not add @type, but instead verify that any type is
```

Comment:
`s/any/the declared type (if any)/` ? `any` is an overloaded term in TypeScript types :-)

Reply:
No longer applies